DB/src/main/java/io/deephaven/db/v2/parquet/ParquetReaderUtil.java

```diff
@@ -265,12 +266,13 @@ public static void readParquetSchema(final String filePath, final ColumnDefiniti
             final LogicalTypeAnnotation logicalTypeAnnotation = column.getPrimitiveType().getLogicalTypeAnnotation();
             final String colName = column.getPath()[0];
             final boolean isGrouping = groupingCols.contains(colName);
-            final String codecName = keyValueMetaData.get(ParquetTableWriter._CODEC_NAME_PREFIX_ + colName);
-            final String codecArgs = keyValueMetaData.get(ParquetTableWriter._CODEC_ARGS_PREFIX_ + colName);
-            String codecType = keyValueMetaData.get(ParquetTableWriter._CODEC_DATA_TYPE_PREFIX_ + colName);
+            final String codecName = keyValueMetaData.get(ParquetTableWriter.CODEC_NAME_PREFIX + colName);
+            final String codecArgs = keyValueMetaData.get(ParquetTableWriter.CODEC_ARGS_PREFIX + colName);
+            String codecType = keyValueMetaData.get(ParquetTableWriter.CODEC_DATA_TYPE_PREFIX + colName);
+            int objectWidth = -1;
```

Comment:
Shouldn't this be `io.deephaven.util.codec.ObjectDecoder#VARIABLE_WIDTH_SENTINEL` (aka `Integer.MIN_VALUE`)?